fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/directory/AgilityContextTest.java

```diff
@@ -494,6 +501,56 @@ void testCloseOnCommitFailure() {
         }
     }
 
+    @Test
+    void testAutoCommitVersionStampOuterSleep() throws InterruptedException {
```

Comment:
Both of these run against `accept`, it would also be good to run them against `apply`.
Probably no need to run them for any other part of the api, since those all call `accept`/`apply`.

Reply:
✅